### PR TITLE
LOG-1329: Setup release-5.2

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* Modifies manifest for release-5.2

cc @vimalk78 

### Links
https://issues.redhat.com/browse/LOG-1341

### CODE REVIEW

File: bundle/manifests/cluster-logging.v5.2.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ metadata:
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
     # The version value is substituted by the ART pipeline
-    olm.skipRange: ">=4.6.0-0 <5.0.0"
+    olm.skipRange: ">=4.6.0-0 <5.3.0"
```
Comment (vimalk78) [inline bundle/manifests/cluster-logging.v5.2.0.clusterserviceversion.yaml:25]: `5.2.0`

File: manifests/5.2/cluster-logging.v5.2.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ metadata:
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
     # The version value is substituted by the ART pipeline
-    olm.skipRange: ">=4.6.0-0 <5.0.0"
+    olm.skipRange: ">=4.6.0-0 <5.3.0"
```
Comment (vimalk78) [inline manifests/5.2/cluster-logging.v5.2.0.clusterserviceversion.yaml:25]: `5.2.0`

File: bundle/manifests/cluster-logging.v5.2.0.clusterserviceversion.yaml (hunk 1 above)
Comment (vimalk78) [reply bundle/manifests/cluster-logging.v5.2.0.clusterserviceversion.yaml:25]: it was `5.0.0` earlier. so `release-5.1` branch needs to update this as well. 
I will open a PR for 5.1

**Comment by jcantrill** [comment]
I dont expect this to pass until openshift/release#18309 merges

**Pull request merged**